README.md

```diff
@@ -62,7 +61,7 @@ The reference's location and format depends on the following scenarios:
         * servicebinding/secret/host: {.status.address}
         * servicebinding/secret/`<binding_property>`: {.status.`<status_property>}` (where `<binding_property>` is any property from the binding schema, and `<status_property>` refers to the path to the correspoding `status` property)
 
-3. Regular k8s Deployment (Ingress, Route, Service, etc)  - An annotation in the corresponding CR that maps the `status` properties to their corresponding binding data. The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
+3. Regular k8s Deployment (Ingress, Route, Service, etc)  - An annotation in the corresponding CR that maps the `status` properties to their corresponding [binding data](#service-binding-schema). The value of this annotation can be specified in either [JSONPath](https://kubernetes.io/docs/reference/kubectl/jsonpath/) or [GO templates](https://golang.org/pkg/text/template/):
```

Comment:
How about binding to a field in a `Secret` or a `ConfigMap` where the `status` field doesn't really have anything useful? 

e.g. `.data` has all the goodies for secret:
```yaml
apiVersion: v1
kind: Secret
metadata:
  name: mysecret
type: Opaque
data:
  username: YWRtaW4=
  password: MWYyZDFlMmU2N2Rm
```

Reply:
I wonder if approach #3 and #4 can be combined into one...

Reply:
yup, great point.   And also `spec`, to be consistent with another part above in the spec.   I like the idea of combining 3 & 4 - I am wondering if `Application` would be a good thing to call out as well, since it's often used to define internal and external apps.

Reply:
Added.  thanks for the suggestion.